### DISCUSSION TITLE
Webdav max file size

### DISCUSSION THREAD

**Discussion by Joly0** [nextcloud/all-in-one#4676] [Questions] [answered]
Hey guys, i have setup nextcloud using nextcloud-aio through unraid. I have also added my nextcloud instance to windows explorer through webdav. When i try to upload larger files (i have tested with a file ~5GB), i get an error stating "file size is to large for target filesystem". Though uploading via webui, i dont get any errors.

Another thing i am also noticing through webdav, is that files are uploading normally, but when they reach the end of the upload, the upload seems to be stuck for quite a while, and then finishes.

**Comment by szaimen** [comment, marked as answer]
Hi, see https://docs.nextcloud.com/server/stable/user_manual/en/files/access_webdav.html#id5
